packages/styles/scss/components/popover/_popover.scss

```diff
@@ -125,6 +125,17 @@ $popover-caret-height: custom-property.get-var(
 
   // Popover content
   .#{$prefix}--popover-content {
+    @include layout.redefine-tokens(
```

Comment:
Should we add a small note here that this is because we redefine the tokens in `Tag`, so we need to reset them to the defaults here?